src/components/draw/DrawStyleDirective.js

```diff
@@ -15,16 +15,48 @@ goog.require('ga_window_service');
   module.directive('gaDrawStyle', function($document, $window, $translate,
       gaGlobalOptions, gaStyleFactory, gaUrlUtils, gaWindow) {
 
+    var findCategoryBySource = function(src, categories) {
+      var regexCss = new RegExp('^(.*?)-24');
+      if (regexCss.test(src)) {
+        return categories[0];
+      }
+      for (var i = 0; i < categories.length; i++) {
+        var c = categories[i];
+        if (c.prefix) {
+          var regexImg = new RegExp('^(.*?)' + c.prefix);
+          if (regexImg.test(src)) {
+            return c
+          }
+        }
+      }
+      return new Error('No category found for source ' + src);
```

Comment:
Don't return an error if you don't manage it. Instead display a msg in the console with `console.error` and return nothing.